wsnet/listen.go

```diff
@@ -143,7 +143,11 @@ func (l *listener) dial(ctx context.Context) (<-chan error, error) {
 		defer close(errCh)
 		for {
 			conn, err := session.Accept()
+			if errors.Is(err, io.EOF) {
```

Comment:
Does this not mean the yamux session is permanently borked?

Reply:
Yea you're right... debugging the error we're not checking now.